ui/admin/app/components/form/credential-store/vault/index.hbs

```diff
@@ -85,119 +89,135 @@
       </InfoField>
     {{/if}}
   {{/if}}
-  <form.input
-    @name='address'
-    @type='text'
+
+  <Hds::Form::TextInput::Field
+    name='address'
     @value={{@model.address}}
-    @label={{t 'resources.credential-store.form.address.label'}}
-    @error={{@model.errors.address}}
-    @helperText={{t 'resources.credential-store.form.address.help'}}
-    disabled={{@model.isSaving}}
-    readonly={{false}}
-    as |field|
+    @isInvalid={{@model.errors.address}}
+    @isRequired={{true}}
```

Comment:
@dvonanderson 
(Note: I am using mirage): I am able to create and edit Vault Credential Store without filling Required fields, so I don't think the required is working?

Reply:
Nevermind my previous comment.

I quickly check this against a Boundary instance, in that case the required works no problem, so I will test this PR using a boundary instance and not vercel. 

I will bring this to tech time. I think we should find a way that using mirage, we fail if a required field is not fill. But I do think, that's a separate topic of this PR.